policyengine_us/parameters/gov/states/wa/tax/income/capital_gains/deductions/charitable/cap.yaml

```diff
@@ -1,12 +1,14 @@
 description: Washington capital gains charitable contribution cap.
 metadata:
   label: Washington capital gains charitable contribution cap
-  name: wa_ltcg_charitable_contributions_cap
   unit: currency-USD
   period: year
   reference:
-    - title: "RCW 82.87.080(2) Charitable donation deduction"
+    - title: RCW 82.87.080(2) Charitable donation deduction
       href: https://app.leg.wa.gov/RCW/default.aspx?cite=82.87.080
+    - title: Capital gains tax | Washington Department of Revenue
+      href: https://dor.wa.gov/taxes-rates/other-taxes/capital-gains-tax
 values:
   0000-01-01: 0
   2022-01-01: 100_000
```

Comment:
check for 2021 values & applicability - if not in effect in 2021 - make an in_effect bool parameter